service/controller/appcatalog/resource/appcatalogentry/resource.go

```diff
@@ -23,10 +23,12 @@ import (
 const (
 	Name = "appcatalogentry"
 
-	apiVersion          = "application.giantswarm.io/v1alpha1"
-	kindAppCatalog      = "AppCatalog"
-	kindAppCatalogEntry = "AppCatalogEntry"
-	maxEntriesPerApp    = 5
+	apiVersion           = "application.giantswarm.io/v1alpha1"
+	communityCatalogType = "community"
+	kindAppCatalog       = "AppCatalog"
+	kindAppCatalogEntry  = "AppCatalogEntry"
+	maxEntriesPerApp     = 5
+	metadataAnnotation   = "application.giantswarm.io/metadata"
```

Comment:
Let's have this in pkg/annotation or apiextensions. There are discussions in sig-operator about having a metadata library instead of using apiextensions. So I think pkg is fine for now.

Reply:
Good catch, I will move this into pkg.